test/utils/setup.ts

```diff
@@ -111,12 +111,8 @@ export const migrationContract = async () => {
     return await hre.ethers.getContractFactory("Migration");
 };
 
-export const migrationContractTo150 = async () => {
-    return await hre.ethers.getContractFactory("Safe150Migration");
-};
-
-export const migrationContractFrom130To141 = async () => {
-    return await hre.ethers.getContractFactory("Safe130To141Migration");
+export const safeMigrationContract = async () => {
+    return await hre.ethers.getContractFactory("SafeMigration");
 };
```

Comment:
I think you can add it to the library deployment scripts directly since we intend to release it

Reply:
Adding to the deploy script would require that singleton and fallback handler addresses are provided in constructor args. This is a generic contract and we have tests that cover 1.4.1 and 1.50 as target versions.

Do you mean we deploy migration SafeMigration instance for each target version?

Reply:
This gets addressed to some extent: https://github.com/safe-global/safe-smart-account/pull/793/commits/cc4a8e0acda724e4a7786c2b396669dbe0736422

Reply:
It does, but we'll still need to deploy it: https://github.com/safe-global/safe-smart-account/issues/781

I can work on it later, but if you add it you'd be able to reuse that part in the tests

Reply:
We should definitely add it to the deploy scripts, but it doesn't have to be in this PR.